.github/workflows/lint.yml

```diff
@@ -42,6 +42,7 @@ jobs:
         run: |
           # this will update docs/source/settings.rst - but will not create html output
           (cd docs && sphinx-build -b "dummy" -d _build/doctrees source "_build/dummy")
+          git update-index --assume-unchanged docs/source/settings.rst
```

Comment:
Huh? That was the primary goal of having CI regenerate docs. So that introducing deviations between code & documentation is pointed out in pull requests. If the default is no longer OS-independent, then the docs should say so.